Client fails to start on Unix due to backslashes in the resource pack path.

```
Unhandled Exception:
System.IO.FileNotFoundException: Specified Zip does not exist: /Users/pieterjan_briers/code/space-station-14/bin/Client/..\..\Resources\ResourcePack.zip
  at SS14.Client.Resources.ResourceManager.LoadResourceZip (System.String path, System.String pw) [0x000b7] in <a2f92a681d9f4d03af4624939cac6c84>:0
  at SS14.Client.Resources.ResourceManager.LoadLocalResources () [0x00001] in <a2f92a681d9f4d03af4624939cac6c84>:0
  at SS14.Client.GameController.Run () [0x0003b] in <a2f92a681d9f4d03af4624939cac6c84>:0
  at SS14.Client.Program.Main () [0x00013] in <a2f92a681d9f4d03af4624939cac6c84>:0
[ERROR] FATAL UNHANDLED EXCEPTION: System.IO.FileNotFoundException: Specified Zip does not exist: /Users/pieterjan_briers/code/space-station-14/bin/Client/..\..\Resources\ResourcePack.zip
  at SS14.Client.Resources.ResourceManager.LoadResourceZip (System.String path, System.String pw) [0x000b7] in <a2f92a681d9f4d03af4624939cac6c84>:0
  at SS14.Client.Resources.ResourceManager.LoadLocalResources () [0x00001] in <a2f92a681d9f4d03af4624939cac6c84>:0
  at SS14.Client.GameController.Run () [0x0003b] in <a2f92a681d9f4d03af4624939cac6c84>:0
  at SS14.Client.Program.Main () [0x00013] in <a2f92a681d9f4d03af4624939cac6c84>:0
```

The problem is that the resource pack path is specified using Windows style backslashes, which obviously doesn't work on Unix like operating systems.

--- a/SS14.Client/ResourceManagment/ResourceManagerZip.cs
+++ b/SS14.Client/ResourceManagment/ResourceManagerZip.cs
@@ -91,7 +91,7 @@ namespace SS14.Client.Resources
         {
             var cfgMgr = _configurationManager;
 
-            cfgMgr.RegisterCVar("res.pack", @"../../Resources/ResourcePack.zip", CVarFlags.ARCHIVE);
+            cfgMgr.RegisterCVar("res.pack", Path.Combine("..","..","Resources","ResourcePack.zip"), CVarFlags.ARCHIVE);
             cfgMgr.RegisterCVar("res.password", String.Empty, CVarFlags.SERVER | CVarFlags.REPLICATED);
 
             string zipPath = path ?? _configurationManager.GetCVar<string>("res.pack");

--- a/SS14.Client/ResourceManagment/ResourceManagerZip.cs
+++ b/SS14.Client/ResourceManagment/ResourceManagerZip.cs
@@ -100,7 +100,7 @@ namespace SS14.Client.Resources
             if (AppDomain.CurrentDomain.GetAssemblyByName("SS14.UnitTesting") != null)
             {
                 string debugPath = "..";
-                zipPath = Path.Combine(zipPath, debugPath);
+                zipPath = Path.Combine(debugPath, zipPath);
             }
 
             zipPath = PathHelpers.ExecutableRelativeFile(zipPath);

--- a/SS14.Client/ResourceManagment/ResourceManagerZip.cs
+++ b/SS14.Client/ResourceManagment/ResourceManagerZip.cs
@@ -99,9 +99,8 @@ namespace SS14.Client.Resources
 
             if (AppDomain.CurrentDomain.GetAssemblyByName("SS14.UnitTesting") != null)
             {
-                string debugPath = "..\\";
-                debugPath += zipPath;
-                zipPath = debugPath;
+                string debugPath = "..";
+                zipPath = Path.Combine(zipPath, debugPath);
             }
 
             zipPath = PathHelpers.ExecutableRelativeFile(zipPath);

--- a/SS14.Shared/Utility/PathHelpers.cs
+++ b/SS14.Shared/Utility/PathHelpers.cs
@@ -1,4 +1,4 @@
-using System.IO;
+﻿using System.IO;
 using System;
 using System.Reflection;
 using System.Collections.Generic;
@@ -22,7 +22,7 @@ namespace SS14.Shared.Utility
         /// </summary>
         public static string ExecutableRelativeFile(string file)
         {
-            return Path.Combine(GetExecutableDirectory(), file);
+            return Path.GetFullPath(Path.Combine(GetExecutableDirectory(), file));
         }
 
         public static string AssemblyRelativeFile(string file, Assembly assembly)

